Examples/ImageRegistrationMethodDisplacement1/ImageRegistrationMethodDisplacement1.cxx

```diff
@@ -180,15 +174,9 @@ int main(int argc, char *argv[])
   R.MetricUseFixedImageGradientFilterOff();
 
   {
-  std::vector<unsigned int> shrinkFactors;
-  shrinkFactors.push_back(3);
-  shrinkFactors.push_back(2);
-  shrinkFactors.push_back(1);
+  std::vector<unsigned int> shrinkFactors { 3, 2, 1 };
```

Comment:
This line is another way to initialize with out the assignment. Please add the assignment for consistency.